agendas/2022/2022-01-06.md

```diff
@@ -132,3 +133,7 @@ This is an open meeting in which anyone in the GraphQL community may attend.
 1. Update on Client Controlled Nullability (10m, Alex Reilly)
    - [Spec PR](https://github.com/graphql/graphql-spec/pull/895)
    - How are decisions typically made when there isn't consensus?
+1. Update on `@defer` & `@stream` (30m, Rob Richard)
+   - [Enforcing delivery order of payloads](https://github.com/robrichard/defer-stream-wg/discussions/17)
+   - [Mutation execution order](https://github.com/robrichard/defer-stream-wg/discussions/1)
+   - [Non-nullable field errors](https://github.com/robrichard/defer-stream-wg/discussions/23)
```

Comment:
@robrichard Do you think it makes sense to add an agenda item for discussing opt-in/out for defer and stream (https://github.com/robrichard/defer-stream-wg/discussions/12)? Or do you think it is too implementation-specific to be discussed in the graphql working group?

Reply:
I was thinking we can can discuss that at the next graphql-js WG  meeting.